ramls/release-encumbrance.raml

```diff
@@ -7,7 +7,7 @@ baseUri: https://github.com/folio-org/mod-finance
 
 documentation:
   - title: Release encumbrance API
-    content: This documents the API calls that release any remaining money encumbered back to the budget's available pool
+    content: This documents the API calls that release any remaining money encumbered back to the budget's available pool; this should only be used by the UI, and only to release a single encumbrance - batch-all-or-nothing should be used otherwise
```

Comment:
Can we have a multiline `content` to avoid long lines?

Reply:
Yes. And I used the occasion to use `<tt>` for the new endpoint.